Remove extra footer

portdeveloper opened the pull request:
## Description

In some resolutions, the mini footer is unclickable due to the og footer coming on top of it.

### Before
https://github.com/BuidlGuidl/abi.ninja/assets/108868128/afba4131-5efa-481f-9d37-6c52ebe3e535


### After

https://github.com/BuidlGuidl/abi.ninja/assets/108868128/f267d8ce-3742-429e-818a-0f20eaaf947e



## Additional Information

- [ ] I have read the [contributing docs](https://github.com/BuidlGuidl/abi.ninja/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [ ] This is not a duplicate of any [existing pull request](https://github.com/BuidlGuidl/abi.ninja/pulls)

## Related Issues

_Closes #{issue number}_

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address:

technophile-04 reviewed (commented):
Nice catch !! Tysm @portdeveloper, a small nitpick 😅 : 

![Screenshot 2024-03-25 at 5 04 55 PM](https://github.com/BuidlGuidl/abi.ninja/assets/80153681/7c69be9d-b5fe-420d-b78b-5740ba33ba89)


I think we lost the margin top on home screen for footer on small devices

portdeveloper commented:
Thanks for the feedback @technophile-04 !

Fixed, also changed the way the padding works on index, increased the bottom padding to make the footer look further apart from the bottom border.

technophile-04 reviewed (approved):
Nicceee, its working great!! Tysm @portdeveloper 🙌!!
